### PR TITLE
Github Actions: Notifies stale issues and pull requests

### PR DESCRIPTION
**Pull request by ucalyptus**
## Marks issues and pull requests that have not had recent interaction.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Stale issue message'
+        stale-pr-message: 'Stale pull request message'
```
Comment (shaz13-socgen) [inline .github/workflows/stale.yml]: LGTM. Can do please make these messages more appealing rather than - "Stale ..."

**Review by shaz13-socgen — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 1 above)
Comment (ucalyptus) [reply .github/workflows/stale.yml]: Sure, on it

**Comment by ucalyptus** [comment]
@shaz13-socgen @shaz13  have a look

**Comment by shaz13** [comment]
Init - Add space after “.” for both messages. LGTM ✅

**Comment by ucalyptus** [comment]
@shaz13 done

**Comment by shaz13-socgen** [comment]
LGTM. Merging PR. Thanks a bunch for your contributions @ucalyptus

**Review by shaz13-socgen — APPROVED** [review]

**Pull request merged**